test/assert.js

```diff
@@ -20,6 +20,18 @@ describe('assert', function () {
     }).to.throw(chai.AssertionError, /this has failed/);
   });
 
+  it('fail', function () {
+    chai.expect(function () {
```

Comment:
Rater than using `chai.expect.to.throw`, I think we can use the same `err()` helper function for these `assert` interface tests that are used in the `expect` and `should` interface tests. Doing so has the advantage of the test making sure implementation frames aren't in the Assertion Error's stack trace.